ucgenerator/main.go

```diff
@@ -0,0 +1,598 @@
+package main
```

Comment:
Should we perhaps add a build tag?

Reply:
Turns out, go walks through imports. If module not imported, it's not included.